remote_function/requirements.txt

```
@@ -1,4 +1,4 @@
-flask>=3.0.2
+flask
```

Is there a minimum version requirement? Was there an issue with the previous min. version? If not, let's keep the previous version:
```suggestion
flask>=3.0.2
```